js-sdk/README.md

```diff
@@ -0,0 +1,65 @@
+# Gitcoin SDK
+
+## Dynamic Widget
+
+### Example
+
+Step 1: Include the JavaScript SDK on your page once, ideally right after the opening body tag.
+
+```html
+<div id="gc-root"></div>
+<script>(function(d, s, id) {
+  var js, gjs = d.getElementsByTagName(s)[0];
+  if (d.getElementById(id)) return;
+  js = d.createElement(s); js.id = id;
+  js.src = "https://unpkg.com/gitcoin-sdk";
+  gjs.parentNode.insertBefore(js, gjs);
+}(document, 'script', 'gitcoin-jssdk'));</script>
+```
+
+Step 2: Place this code wherever you want the plugin to appear on your page.
+
+```html
+<div class="gitcoin-widget"
+  data-limit="2"
+  data-order-by="-expires_date"
+  data-orginisation="MetaMask"
+  data-repository="metamask-extension"
+></div>
+```
+
+### Autoloading
+
+Importing the SDK into your application will attempt to autoload the widget by searching for '.gitcoin-widget' selectors
+
+```javascript
+import 'gitcoin-sdk';
+```
+or
+```javascript
+require('gitcoin-sdk');
+```
+
```

Comment:
should we include a webified version in the documentation?

Reply:
Hello @owocki,

The library should be targeted for the web, the simplest way to use it as a webmasteris by following the initial Example, (Part1 & Part2), this follows the same method used by Facebook and Twitter for inclusion of their own widgets. 

For example: https://developers.facebook.com/docs/plugins/like-button

Reply:
ah you are right.. thanks

Reply:
> ah you are right.. thanks

The documentation isn't clear enough about that, I'll make some updates to it shortly.